controllers/userController.js

```diff
@@ -9,41 +9,38 @@ module.exports = {
   addUser: async (req, res) => {
     try {
       const {
-        roleId,
         username,
         password,
         confirmPassword,
         emailId,
         firstName,
         lastName,
+        roleId,
       } = req.body;
-      if (password === confirmPassword) {
+      if (password == confirmPassword) {
         const salt = await bcrypt.genSalt(10);
         const securePassword = await bcrypt.hash(password, salt);
 
-        await User.create({
-          roleId,
+        const createUser = await User.create({
           username,
           password: securePassword,
           emailId,
           firstName,
           lastName,
+          roleId,
         });
-        res.status(200).send({
-          message: "data saved successfully",
-        });
+        res.status(200).send(createUser);
```

Comment:
if you are changing the response, change it like
.send({
message: "User registered successfully",
data: createUser 
})
@akaashinno